### ISSUE TITLE
Accessibility Audit: Current image index and number of images is not clear for users with screen readers 

### ISSUE THREAD

**Issue by brittnylapierre** [ProjectMirador/mirador#3886]
Description: An accessibility audit of Mirador was performed. It was found that the number of items and the current image index was not clear to users with screen readers. 

Supporting Comments:
- LIMITED SIGHT – Windows 10/NVDA 2022.4/ Chrome - NVDA doesn't clearly announce the selected item. 
- DEAF-BLIND – Windows 10/JAWS 2019/Focus 14 braille/Chrome - Tabbing necessary to read the current index. JAWS reported being on page 4 when tabbing, but arrow indicated page 1 
- LIMITED SIGHT – Apple Mac Mini M1 2000/MacOS 13.4.1/27” external monitor/Increased contrast /1280 by 72px/zooming - No clear indication of the current image. 
- Blind – JAWS 2024 HOME/EDGE/WINDOWS 10 - Could not find the current page indication 
- DEAF-BLIND – iPhone 13 Pro Max/iOS 17.2/Focus 14 Blue 4th gen/VoiceOver/Safari - Unable to find current page index. 
- DEAF – BLIND – Ipad Pro 9/iPadOS 17.2/Focus 40 Blue 4th gen/VoiceOver/Safari - Could not find a page index. 
- DEAF – BLIND – Ipad Pro 9/iPadOS 17.2/Focus 40 Blue 4th gen/VoiceOver/Safari - Unsure about the number of items and thumbnails in the viewer. 
- DEAF-BLIND – Windows 11/laptop/JAWS 2020/PacMate 20 braille/Chrome - Discrepancy in JAWS reporting page numbers, requiring tabbing to get accurate information. 
- DEAF-BLIND – Windows 11/laptop/JAWS 2020/PacMate 20 braille/Chrome - Difficulty determining the number of items or thumbnails present. 
- DEAF-BLIND – iPhone 13 Pro Max/iOS 17.2/Focus 14 Blue 4th gen/VoiceOver/Safari - Unsure about the number of items and thumbnails in the viewer
